Simplenote/Classes/SPTagView.m

```diff
@@ -357,8 +356,9 @@ - (void)processTextInFieldToTag {
 
     if (name.length > 0 && [self.tagDelegate tagView:self shouldCreateTagName:name] && !containsEmailAddress) {
         
-        [self newTagPillWithString:name];
-        [self.tagDelegate tagView:self didCreateTagName:name];
+        NSString *cleanedName = [name stringByTrimmingCharactersInSet:[NSCharacterSet punctuationCharacterSet]];
```

Comment:
as far as I know this is overly constrictive and will remove tags with legitimate characters. so far we have only disallowed spaces and the comma